dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -459,7 +461,12 @@ public String messageLogFormat(LogRecord logRecord, String formattedVerboseMsg)
         // This is a very light trace format, based on enhanced:
         StringBuilder sb = new StringBuilder(256);
         String sym = getMarker(logRecord);
-        String name = nonNullString(logRecord.getLoggerName(), logRecord.getSourceClassName());
+        String name = null;
+        if (logRecord.getLoggerName() != null & (logRecord.getLoggerName().equals(SYSOUT) ||
+                                                 logRecord.getLoggerName().equals(SYSERR)))
+            name = nonNullString(logRecord.getLoggerName(), null);
```

Comment:
you already know getLoggeName != null here.  could just say name=logRecord.getLoggerName()